ksqldb-engine/src/main/java/io/confluent/ksql/engine/SandboxedExecutionContext.java

```diff
@@ -162,19 +162,23 @@ public ExecuteResult execute(
       final ServiceContext serviceContext,
       final ConfiguredKsqlPlan ksqlPlan
   ) {
-    final ExecuteResult result = EngineExecutor.create(
-        engineContext,
-        serviceContext,
-        ksqlPlan.getConfig()
-    ).execute(ksqlPlan.getPlan());
+    try {
```

Comment:
Where was the error being thrown before?

Reply:
This is the validation context. SO if the query fails due to an error it would show up here. We want to close the validation runtimes no matter what